### PR TITLE
feat: new callout, update implementation status and tweaks

### PR DESCRIPTION
**Pull request by hugomrdias**
feat: add callout container
fix: links and remove issue from features
feat: complete implementation-status overhaul

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -439,17 +430,22 @@ module.exports = {
...
       {
         host: DEPLOY_DOMAIN
       }
+    ],
+    [
+      'vuepress-plugin-container',
+      {
+        type: 'callout',
+        defaultTitle: ''
+      }
```
Comment (cwaring) [inline docs/.vuepress/config.js:448]: this plugin is enabled in the parent theme so we shouldn't require it here too?

Comment (hugomrdias) [reply docs/.vuepress/config.js:448]: to create a new container we need to add another array here

Comment (hugomrdias) [reply docs/.vuepress/config.js:448]: https://github.com/vuepress/vuepress-plugin-container#usage

Comment (cwaring) [reply docs/.vuepress/config.js:448]: sorry, what I was meaning is could we reuse one of the existing blocks? https://vuepress.vuejs.org/guide/markdown.html#custom-containers

Comment (cwaring) [reply docs/.vuepress/config.js:448]: just noticed they print the type too so wouldn't be suitable. ok - let's go this route!

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -370,16 +370,13 @@ module.exports = {
     }
   },
   plugins: [
-    ['@vuepress/plugin-back-to-top', true],
+    '@vuepress/back-to-top',
```
Comment (cwaring) [inline docs/.vuepress/config.js:373]:
```suggestion
    '@vuepress/plugin-back-to-top',
```

`@vuepress/back-to-top` doesn't exist

**Comment by cwaring** [comment]
This is good but please let's keep any further style and functionality changes out of this effort until we get to live. Time is limited atm any this work is extra to review.

**Pull request merged**